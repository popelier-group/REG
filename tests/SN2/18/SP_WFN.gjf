# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -1.05782800   -0.00006100    0.00018900
 H                 -0.91093700   -0.90348700   -0.56277300
 H                 -0.91091100    0.93920300   -0.50068100
 H                 -0.91101200   -0.03591600    1.06407500
 Br                 1.63180900    0.00000800   -0.00001800
 Cl                -3.24092600    0.00000900   -0.00004300

frame.wfn
