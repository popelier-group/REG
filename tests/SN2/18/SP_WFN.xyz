
6

C        0.000010   -0.012654    0.941968
H       -0.574954   -0.906427    0.783011
H       -0.488230    0.935103    0.807620
H        1.063348   -0.060766    0.794690
Br        0.000294    0.023371   -1.747427
Cl       -0.000619   -0.041763    3.124872
