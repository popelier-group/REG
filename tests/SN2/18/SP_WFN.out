 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.18/Gau-16982.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.18/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe PID=     16983.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -1.05783  -0.00006   0.00019 
 H                    -0.91094  -0.90349  -0.56277 
 H                    -0.91091   0.9392   -0.50068 
 H                    -0.91101  -0.03592   1.06408 
 Br                    1.63181   0.00001  -0.00002 
 Cl                   -3.24093   0.00001  -0.00004 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.057828   -0.000061    0.000189
      2          1           0       -0.910937   -0.903487   -0.562773
      3          1           0       -0.910911    0.939203   -0.500681
      4          1           0       -0.911012   -0.035916    1.064075
      5         35           0        1.631809    0.000008   -0.000018
      6         17           0       -3.240926    0.000009   -0.000043
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.074561   0.000000
     3  H    1.074557   1.843736   0.000000
     4  H    1.074567   1.843723   1.843724   0.000000
     5  Br   2.689637   2.756547   2.756479   2.756723   0.000000
     6  Cl   2.183098   2.561605   2.561580   2.561667   4.872735
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.000010   -0.012654    0.941968
      2          1           0       -0.574954   -0.906427    0.783011
      3          1           0       -0.488230    0.935103    0.807620
      4          1           0        1.063348   -0.060766    0.794690
      5         35           0        0.000294    0.023371   -1.747427
      6         17           0       -0.000619   -0.041763    3.124872
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         147.5157746           0.8492443           0.8492442
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       171.0756608002 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  4.78D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147513.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 EnCoef did   100 forward-backward iterations
 SCF Done:  E(RB3LYP) =  -3074.39089722     A.U. after   12 cycles
            NFock= 12  Conv=0.24D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.58713-101.35754 -62.24028 -56.05857 -56.05773
 Alpha  occ. eigenvalues --  -56.05773 -10.09012  -9.27055  -8.44915  -7.03281
 Alpha  occ. eigenvalues --   -7.02673  -7.02673  -6.27388  -6.27102  -6.27102
 Alpha  occ. eigenvalues --   -2.38593  -2.38514  -2.38514  -2.38289  -2.38289
 Alpha  occ. eigenvalues --   -0.63341  -0.55408  -0.51193  -0.30678  -0.30678
 Alpha  occ. eigenvalues --   -0.20051  -0.13991  -0.13991  -0.08154  -0.08023
 Alpha  occ. eigenvalues --   -0.08023
 Alpha virt. eigenvalues --    0.10103   0.11511   0.15214   0.15215   0.16585
 Alpha virt. eigenvalues --    0.17249   0.19070   0.19070   0.20844   0.21182
 Alpha virt. eigenvalues --    0.21182   0.22730   0.22730   0.24731   0.29402
 Alpha virt. eigenvalues --    0.31563   0.31564   0.37167   0.39362   0.54522
 Alpha virt. eigenvalues --    0.54522   0.55953   0.57332   0.58877   0.58877
 Alpha virt. eigenvalues --    0.62517   0.62517   0.69181   0.77259   0.77260
 Alpha virt. eigenvalues --    0.80063   0.81891   0.85239   0.85239   0.85781
 Alpha virt. eigenvalues --    0.85781   0.92801   1.04748   1.04749   1.06043
 Alpha virt. eigenvalues --    1.06043   1.09751   1.17545   1.23317   1.23318
 Alpha virt. eigenvalues --    1.27352   1.53323   1.63453   1.63454   1.73643
 Alpha virt. eigenvalues --    1.95256   1.95257   1.97892   2.15970   2.15970
 Alpha virt. eigenvalues --    2.22812   2.22812   2.31999   2.44885   2.44885
 Alpha virt. eigenvalues --    2.53352   2.62351   2.62351   2.67261   2.75686
 Alpha virt. eigenvalues --    2.75687   3.04517   3.20058   3.85848   3.85849
 Alpha virt. eigenvalues --    4.36802   4.36803   4.37734   4.37734   4.55532
 Alpha virt. eigenvalues --    6.76563   7.77558   7.77562   7.84975   9.84778
 Alpha virt. eigenvalues --   24.03770  26.00708  26.00709  26.32672  48.07328
 Alpha virt. eigenvalues --  215.82798 289.98276 289.98277 290.056081020.83836
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.232196   0.420712   0.420716   0.420701  -0.008549  -0.074447
     2  H    0.420712   0.512659  -0.034582  -0.034594  -0.019266  -0.031293
     3  H    0.420716  -0.034582   0.512655  -0.034591  -0.019273  -0.031293
     4  H    0.420701  -0.034594  -0.034591   0.512668  -0.019247  -0.031291
     5  Br  -0.008549  -0.019266  -0.019273  -0.019247  35.811003   0.029080
     6  Cl  -0.074447  -0.031293  -0.031293  -0.031291   0.029080  17.513257
 Mulliken charges:
               1
     1  C   -0.411328
     2  H    0.186365
     3  H    0.186369
     4  H    0.186355
     5  Br  -0.773748
     6  Cl  -0.374012
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C    0.147760
     5  Br  -0.773748
     6  Cl  -0.374012
 Electronic spatial extent (au):  <R**2>=           1139.0303
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              0.0005    Y=              0.0010    Z=             -0.0874  Tot=              0.0874
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -44.0472   YY=            -44.0538   ZZ=            -82.0118
   XY=             -0.0001   XZ=              0.0074   YZ=              0.5074
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             12.6571   YY=             12.6504   ZZ=            -25.3075
   XY=             -0.0001   XZ=              0.0074   YZ=              0.5074
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              1.0597  YYY=              0.6547  ZZZ=            -98.4858  XYY=             -1.0499
  XXY=              0.0188  XXZ=            -12.5943  XZZ=              0.0136  YZZ=              0.9799
  YYZ=            -12.6016  XYZ=             -0.0142
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=            -72.8816 YYYY=            -73.1270 ZZZZ=          -1644.4418 XXXY=             -0.0116
 XXXZ=              0.8588 YYYX=              0.0280 YYYZ=              9.0797 ZZZX=              0.1646
 ZZZY=             12.0368 XXYY=            -24.3312 XXZZ=           -248.0515 YYZZ=           -248.1275
 XXYZ=              2.8870 YYXZ=             -0.6943 ZZXY=             -0.0208
 N-N= 1.710756608002D+02 E-N=-7.695266827272D+03  KE= 3.069971350747D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE529\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\V69787FF\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-1.057828,-0.000061,0.000189\H,0,-0.910937,-0.903487,-0.562773\H,0
 ,-0.910911,0.939203,-0.500681\H,0,-0.911012,-0.035916,1.064075\Br,0,1.
 631809,0.000008,-0.000018\Cl,0,-3.240926,0.000009,-0.000043\\Version=E
 S64L-G16RevC.01\State=1-A\HF=-3074.3908972\RMSD=2.364e-09\Dipole=0.034
 3957,-0.0000545,0.0001733\Quadrupole=-18.8205607,9.410331,9.4102297,0.
 0000795,-0.0003866,-0.0000108\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 THE WORLD OF CHEMICAL REACTIONS IS LIKE A STAGE,
 ON WHICH SCENE AFTER SCENE IS CEASELESSLY PLAYED.
 THE ACTORS ON IT ARE THE ELEMENTS.
   -- CLEMENS WINKLER   BER. 30,13(1897)
 (DISCOVERER OF GERMANIUM, FEB 6, 1886)
 Job cpu time:       0 days  0 hours  0 minutes 27.1 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 13.9 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:49:26 2020.
